bin/server.js

```diff
@@ -81,7 +81,7 @@ const onconnection = conn => {
     closed = true
   })
   conn.on('message', /** @param {object} message */ message => {
-    if (typeof message === 'string') {
+    if (typeof message === 'string' || Buffer.isBuffer(message)) {
```

Comment:
Don't we have to convert the buffer to a string first?

Maybe we should just add this:

```js
if (Buffer.isBuffer(message)) {
  message = message.toString()
}
```